### PR TITLE
docs: Fix example for bindCallback

### PR DESCRIPTION
**Pull request by ondrasej**
docs: Fix the example for bindCallback.

**Description:**

The example, as it was, crashes with `Error: cb is not a function` on line 5. The new example demonstrates how arguments are forwarded or added.

**Related issue (if exists):**

### CODE REVIEW

**Review by jakovljevic-mladen — APPROVED** [review]
LGTM. Thanks for fixing this.

**Comment by ondrasej** [comment]
Thanks for approving!

Is there something I need to do to merge, or just wait for someone who has merge rights?

**Comment by benlesh** [comment]
Merged into 7.x and master.

**Pull request merged**
